catalogue_pipeline/transformer/src/test/scala/uk/ac/wellcome/platform/transformer/SierraTransformerFeatureTest.scala

```diff
@@ -65,21 +63,21 @@ class SierraTransformerFeatureTest
                 val sourceIdentifier = SourceIdentifier(
                   identifierType = IdentifierType("sierra-system-number"),
                   ontologyType = "Work",
-                  value = "b10010014"
+                  value = sierraTransformable.sierraId.withCheckDigit(
```

Comment:
mmmh...isn't this a bit testing the prod code with the same code? Have we kept a test somewhere where we explicitly check the output value against a hardcoded string?

Reply:
See `SierraRecordNumberTest`